set_point_generator_amplitude : 5.0